pulsar-functions/instance/src/test/java/org/apache/pulsar/functions/windowing/WindowFunctionExecutorTest.java

```diff
@@ -218,24 +218,14 @@ public void testExecuteWithLateTupleStream() throws Exception {
     @Test
     public void testSettingSlidingCountWindow() throws Exception {
         final Object[][] args = new Object[][]{
-                {-1, 10},
```

Comment:
These test cases are now part of the WindowConfigUtilsTest. Since the validation logic has been moved from executor to the central place, we can't do these cases in this test,.

Reply:
This is confusing to me since now that is also a test "testSettingSlidingCountWindow" in WindowConfigUtilsTest that is kind of doing to same thing

Reply:
The one in WindowConfigUtilsTest is just doing sanity checking while the checks here are doing functionality check

Reply:
We have to separate them? We have positive test cases in one class and negative test cases in another.  That is really confusing

Reply:
I guess I see why you separated them because the validation from WindowedExecutor is removed

Reply:
Yes, now there is one place for argument validation and another place for actual functional checking